frontend/src/patientApp/timeOfTest/ToS.tsx

```diff
@@ -59,8 +59,8 @@ function ToS() {
         i18nKey="testResult.tos.document.privacy.p0"
         parent="p"
         components={[
-          <a href="https://www.cms.gov/Research-Statistics-Data-and-Systems/Computer-Data-and-Systems/Privacy/PrivacyActof1974.html">
-            click here
+          <a href="https://www.hhs.gov/foia/privacy/index.html">
+            this content is replaced
```

Comment:
not sure if we have a preference for what to put here
the inner contents of the component [gets replaced](https://github.com/i18next/react-i18next/issues/1090#issuecomment-615276404) but the linter won't let me leave it empty